### PR TITLE
pa where, pa comprehension: Moved the packages to GitHub

### PR DESCRIPTION
**Pull request by cakeplus**

### CODE REVIEW

File: packages/pa_where/pa_where.0.4/opam [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 build: [
-  [make "build"]
-  [make "install"]
+  ["ocaml" "setup.ml" "-configure"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-install"]
 ]
-remove: [["ocamlfind" "remove" "pa_where"]]
-depends: ["camlp4" "ocamlfind"]
+remove: [["ocaml" "remove" "pa_where"]]
```
Comment (yallop) [inline packages/pa_where/pa_where.0.4/opam]: Should this be 

```
remove: [["ocamlfind" "remove" "pa_where"]]
```
?

Comment (cakeplus) [inline packages/pa_where/pa_where.0.4/opam]: Thank you for spotting this. Fixed.

**Comment by yallop** [comment]
Thanks!

**Comment by Chris00** [comment]
P.S. You may want to use `oasis2opam` to easily generate the opam files in the future.

**Comment by cakeplus** [comment]
Thanks for the suggestion. I didn't know about `oasis2opam`. Will try using it right now, for an upcoming package.

**Pull request merged**